exercise-1.js

```diff
@@ -1,143 +1,288 @@
+                }
+                else{
+                    console.log(i)
+                }
+            }
+        }
+        tekCamp()
+    }
 
 
 
+        //2 problem
         /************************************************************************************/
         //Write a function that converts the current temperature from Fahrenheit to Celsius. 
 
         //your code...
-        function farenheitCelsius() {
-
+    {
+        function farenheitCelsius(farenheit) {
+            let celcius = (farenheit - 32) * (5 / 9);
+            return celcius;
         }
+        farenheitCelsius(50);
+    }
 
 
 
-
+        //3 problem
         /************************************************************************************/
         //Write a function that converts the Celsius temperature back to Fahrenheit. 
 
         //your code...
-        function celsiusFarenheit() {
-
+    {
+        function celsiusFarenheit(celsius) {
+            let farenheit = (celsius * 1.8) + 32;
+            return farenheit;
         }
+        celsiusFarenheit(80)
+    }
 
 
 
 
 
+        //4 problem
         /************************************************************************************/
         // Write a function to determine if someone is old enough to vote.  The function should return a boolean of true or false.
 
         //your code...
+    {
         function canVote(age) {
-            return false;
+            age >= 18?true:false;
         }
 
+        canVote(17);
+    }
 
 
 
 
-
-
+        //5 problem
         /************************************************************************************/
         // Write a function that converts a string to an array. It should return an array.  
 
         //your code...
+    {
         function strToArr() {
-            return [];
+            const str = 'tekCamp awesome!';
+            return str.split();
         }
+        strToArr();
+    }
 
 
 
-
-
-
-
+        //6 problem
         /************************************************************************************/
         // Write a function that reverses your telephone number.  It should return the reversed telephone number.
 
         //your code...
-        function reversePhone(number) {
-            
+    {
+        function reversePhone(nr) {
+            let reversed = 0;
+            while(nr > 0){
+                reversed = (reversed * 10) + (nr % 10);
+                nr = Math.floor(nr / 10);
+            }
+            return reversed;
         }
+        reversePhone(4254992269)
+        // console.log(reversePhone(4254992269))
+    }
 
 
 
-
-
+        //7 problem
         /************************************************************************************/
         // Write a function that returns a car object using some given info about your car. Required inputs are the make, model, year, and color.
+        
+         //your code...
+    {
+        function carFunct(make, model, year, color) {
```

Comment:
also, it may make sense to use a 'verb' for function names, because a function "does something". 
for example : constructCar();   
Or the convention of Pascal casing for constructor functions in JavaScript. 
`function Car()`